mod/wikirate_utility/set/self/search/categories.haml

```diff
@@ -0,0 +1,9 @@
+= nest [:wikirate_company, :search], view: :titled,
+                                     title: "Companies",
+                                     items: { view: :info_bar }
+= nest [:wikirate_topic, :search], view: :titled,
+                                   title: "Topics",
+                                   items: { view: :info_bar }
+= nest [:metric, :search], view: :titled,
+                           title: "Metrics",
+                           items: { view: :info_bar }
```

Comment:
not really related to projects.  This is about getting the "row" class out of the search results, because that was not playing well with the bar views (made them not full width)